samples/features/sql-big-data-cluster/connectivity/webhdfs-java-client/README.md

```diff
@@ -0,0 +1,64 @@
+# Building this example in Linux
+
+## Prepare dependencies jar into work dir
+
+bash ./prepare.sh
+## Building jars
+
+bash ./build.sh
+
+# Running this sample
+
+The App.java has hardcoded values in it mentioned in the sections below. Those should be pointed to the right values.
```

Comment:
It might be helpful to parameterize the hardcoded values.

Reply:
Agreed ( including the file name mentioned in one of the comments ), thanks